wiki/People/Users_with_unique_titles/de.md

```diff
@@ -1,3 +1,7 @@
+---
+outdated_translation: true
```

Comment:
Please don't forget to add `outdated_since: [hash]` to all translations as well.

Reply:
where would i find the hash? is it 0c632eb836db2ff8a3ae85d3e658a6e983979d6f from my original pr

Reply:
should be done